code/game/turfs/unsimulated/snow.dm

```diff
@@ -89,30 +89,46 @@
 
 /turf/unsimulated/floor/snow/Exited(atom/A, atom/newloc)
 	..()
-	if(istype(A,/mob/living/carbon/human))
-		var/mob/living/carbon/human/H = A
-		if(snowprint_parent && snowballs && !H.flying)
-			if(!H.locked_to && !H.lying) //Our human is walking or at least standing upright, create footprints
-				snowprint_parent.AddSnowprintGoing(H.get_footprint_type(), H.dir)
-			else //Our human is down on his ass or in a vehicle, create tracks
-				snowprint_parent.AddSnowprintGoing(/obj/effect/decal/cleanable/blood/tracks/wheels, H.dir)
+	
+	var/mob/living/carbon/human/H
+	if(ismecha(A))
+		var/obj/mecha/mecha = A
+		if(ishuman(mecha.occupant))
```

Comment:
just look for mobs, since you can have bagged / gay baby caged mobs

Reply:
Aight does this iteration work?